validator/src/cli.rs

```diff
@@ -372,8 +372,9 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                        [default: ask --entrypoint, or 127.0.0.1 when --entrypoint is not provided]"),
         )
         .arg(
-            Arg::with_name("tpu_host_addr")
-                .long("tpu-host-addr")
+            Arg::with_name("public_tpu_addr")
+                .long("public-tpu-address")
+                .alias("tpu-host-addr")
```

Comment:
do we actually need to change this in this PR?
seems like a separate change, and unrelated to adding the admin rpc command.

Reply:
@behzadnouri
No, we do not.
Initially, the [suggestion](https://github.com/solana-labs/solana/pull/30452#discussion_r1131851386) was made by @joncinque to rename the argument from `tpu-host-addr` to `tpu-host-address`, similar to the new subcommand `set-tpu-host-address`. 
Then, it was [suggested](https://github.com/solana-labs/solana/pull/30452#issuecomment-1475316593) by you to rename the subcommand to `set-public-tpu-addr`.
Following [some research](https://github.com/solana-labs/solana/pull/30452#issuecomment-1490188396), I renamed it to `set-public-tpu-address`, and as the argument was previously changed in this PR, I also changed it to `public-tpu-address`.

So, I will revert `tpu-host-addr` renaming and create a separate PR